graphql-dgs-codegen-core/src/main/kotlin/com/netflix/graphql/dgs/codegen/generators/java/DataTypeGenerator.kt

```diff
@@ -360,6 +381,35 @@ abstract class BaseDataTypeGenerator(
 
     private fun addField(fieldDefinition: Field, javaType: TypeSpec.Builder) {
         addFieldWithGetterAndSetter(fieldDefinition.type, fieldDefinition, javaType)
+        // Generate for all fields except non-nullable without default values
+        if (!(!fieldDefinition.isNullable && fieldDefinition.initialValue == null)) {
```

Comment:
Same comment here as before, I would avoid that "double negative" and write this as:

```kotlin
if (fieldDefinition.isNullable && fieldDefinition.initialValue == null) {
   // ...
}
```

Reply:
Updated same as above